concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme/05 Using SCSS.md

```diff
@@ -0,0 +1,47 @@
+You can import the theme bundle from the `bundles` folder from the `devextreme` package in the root `.scss` file:
```

Comment:
First, I'm going to question your article usage in this scenario:

> You can import **the** theme bundle from ...

What theme bundle? You haven't declared it anywhere else prior to this point. This is the very first line in this topic.

Second... From from

> You can import the theme bundle **from** the `bundles` folder **from** the `devextreme` package.

On possible alternative is "in", however, this change would leave us with "in in" instead of "from from", which is hardly better. However, there is a way to resolve this scenario. There's always a way.

```suggestion
You can import a theme bundle from the `bundles` folder in the `devextreme` package found in the root `.scss` file:
```

OR (there are different interpretations) -

```suggestion
You can import a theme bundle from the `bundles` folder of the `devextreme` package in the root `.scss` file:
```

OR maybe you'll be able to come up with something way better than that altogether...